app/views/posts/index.html.erb

```diff
@@ -18,7 +18,8 @@
             <%= hidden_field_tag(:user_id, current_user.id) %>
             <%= hidden_field_tag(:post_id, post.id) %>
             <% already_liked = post.likes.any?{ |like| like.user_id == current_user.id } %>
-            <%= submit_tag "Like (#{post.likes.count})", :disabled => already_liked %>
+            <%= submit_tag "Like", :disabled => already_liked, :class => "like_button", :onclick => "return like_count(#{post.id}, #{post.likes.count})" %>
```

Comment:
Agreed and tried this originally but as the event is working on in an array I couldn't get it to work so that each button/count was unique and being listened to by the function. Tried putting the eventListener into its own for loop, and thus creating a listener on each array instance, and almost got it to work but didn't want to know on first post and seemed overkill. I also tried to call on class, using the id to identify post and go from there but couldn't pull in the post.likes.count in any way. Even tried pushing posts array through, via a json conversion, which worked apart from it wouldn't connect to the relational likes model... 

Essentially I can find no other method that will allow me to access both a unique identifier for which post it is and lastly pull in the likes count to push, or manually increase to one, at the DOM level. It would be good to get some direction on this please.